### PR TITLE
Hotfix: added outside click functionality #246

### PR DESCRIPTION
**Pull request by giteshk11**
# Issue Being Addressed
#246 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[x] Bug Fix
[ ] Refactor
[ ] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?
Before there was no way to close the overflow menu when clicked anywhere on the screen.
Added the v-outside-click library which fixed the problem.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Log in with your account
2. Go to the client's page
3. Click on the overflow menu first
4. Click on edit profile 
5. The overflow menu should not be visible.

### CODE REVIEW

**Review by paulchrisluke — APPROVED** [review]

File: src/layouts/WithTitleBarLayout.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="page min-h-screen flex flex-col text-white text-opacity-medium">
     <header
       class="bg-primary z-20 shadow-4dp flex flex-row items-center p-4 sticky top-0"
+      v-click-outside="onClickOutside"
```
Comment (mort3za) [inline src/layouts/WithTitleBarLayout.vue:5]: This is not related to a specific layout, we should use it in a overflowMenu component.

Comment (mort3za) [reply src/layouts/WithTitleBarLayout.vue]: I meant this v-click-outside shouldn't be inside a layout file at all. We need a new vue component for that overflow menu.

Comment (mort3za) [reply src/layouts/WithTitleBarLayout.vue]: but it's ok to merge if it's a hotfix.

Comment (giteshk11) [reply src/layouts/WithTitleBarLayout.vue]: oh okay,I'll change it to a new component after I finish what I have right now.
